share/instruments/instrumentsxml.h

```diff
@@ -1502,3 +1527,4 @@ QT_TRANSLATE_NOOP3("InstrumentsXML", "D Vne.", "D Violone"),
 QT_TRANSLATE_NOOP("InstrumentsXML", "arco"),
 QT_TRANSLATE_NOOP("InstrumentsXML", "pizzicato"),
 QT_TRANSLATE_NOOP("InstrumentsXML", "tremolo"),
+QT_TRANSLATE_NOOP("OrderXML", "Orchestra"),
```

Comment:
`QT_TRANSLATE_NOOP("OrderXML", "Big Band"),` seems missing

Reply:
;Correct, I didn't run `generateTs.py` after adding the score order. Mainly because more orders will be added :wink: